### PR TITLE
Add testcase for vnc port 5900

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

File: libvirt/tests/cfg/graphics/graphics_functional.cfg [hunk 1]
```diff
@@ -19,6 +19,8 @@
                             spice_port = 0
                         - port_65535:
                             spice_port = 65535
+                        - port_5900:
```
Comment (chunfuwen) [inline libvirt/tests/cfg/graphics/graphics_functional.cfg:22]: @dzhengfy 
@yafu-1 
I assume there is no plan to automate new spice case as per agreement with feature team since we don't have further bandwidth to maintain deprecated feature  new automation cases

Comment (yafu-1) [reply libvirt/tests/cfg/graphics/graphics_functional.cfg]: @chunfuwen 
Yes, you are right. I deleted the case for spice. 
Thanks.

**Comment by qiankehan** [comment]
Just curious about what's the difference between the existing port_65535 and this port_5900. It's enough to cover just a port of a transmission type

**Comment by yafu-1** [comment]
> Just curious about what's the difference between the existing port_65535 and this port_5900. It's enough to cover just a port of a transmission type
port_5900 and port_65535 are the boundary value for vnc port. I think we need to test both of them.

**Review by dzhengfy — COMMENTED** [review]
@yafu-1 I removed some unnecessary information in PR description field.
Please help provide your test result even though the change is minor. Thanks.

**Comment by yafu-1** [comment]
> @yafu-1 I removed some unnecessary information in PR description field.
> Please help provide your test result even though the change is minor. Thanks.

# avocado run graphics.functional.positive_tests.vnc_only.no_autoport.port_5900 --vt-type libvirt
WARNING:root:No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
WARNING:root:No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 38147f8d6a25ab1176690a2dc983356a7b1b0eba
JOB LOG    : /root/avocado/job-results/job-2021-04-12T22.48-38147f8/job.log
 (1/1) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.vnc_only.no_autoport.port_5900: PASS (9.84 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 10.47 s

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by chunfuwen — APPROVED** [review]
lgtm
